pkg/xds/envoy/routes/reset_tags_configurer.go

```diff
@@ -0,0 +1,19 @@
+package routes
+
+import envoy_api_v2 "github.com/envoyproxy/go-control-plane/envoy/api/v2"
+
+// ResetTagsHeader adds TagsHeader to the RequestHeadersToRemove list. TagsHeader is planned to be used internally,
```

Comment:
`ResetTagsHeader adds x-kuma-tags header to the RequestHeadersToRemove list. x-kuma-tags header is planned to be used internally`.